src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -398,13 +404,18 @@ internal bool ShouldRestartWorkerChannel(string runtime, bool isWebHostChannel,
             return string.Equals(_workerRuntime, runtime, StringComparison.InvariantCultureIgnoreCase) && (isWebHostChannel || isJobHostChannel);
         }
 
-        private async Task RestartWorkerChannel(string runtime)
+        internal async Task RestartWorkerChannel(string runtime)
```

Comment:
Strange that this is called "Restart" when it's actually just starting a new channel. Should we rename this to make it clearer?